plugins/installer/override/override.php

```diff
@@ -336,10 +336,8 @@ private function saveOverrides($pks)
 		{
 			$insertQuery->clear('values');
 
-			$tz = new \DateTimeZone(Factory::getApplication()->get('offset'));
 			$date = new Date('now');
```

Comment:
@laoneo If I am right the timezone of `$date = new Date('now');`
is UTC as this is the default time zone.

Reply:
I guess so too.